### PR TITLE
Upgrade Travis' Ubuntu release

### PR DESCRIPTION
**Pull request by szepeviktor**
> stock versions of pypy provided by Travis CI don't work with cryptography 1.0

http://packages.ubuntu.com/trusty/pypy

### CODE REVIEW

**Comment by szepeviktor** [comment]
pypy3 succeeded: https://travis-ci.org/Backblaze/B2_Command_Line_Tool/builds/171694707

**Comment by szepeviktor** [comment]
We have Jython failure: https://travis-ci.org/Backblaze/B2_Command_Line_Tool/builds/171694707

**Comment by ppolewicz** [comment]
One of the problems here might be related to logfury is shipped. I'll try to fix it.

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ notifications:
   slack:
     secure: naix0bLa/50X1HIXwLtkUC/YH9Iurs18eVy/mhfrfD2YFd7QjUGRz24SQU2r0ev0Qh3CUaH9MJoAUbu4oaXGB1VXzZkV7OIGimZe4gcWJuzItnVhyrQJKpcpC7yafdl7uDA0wjWs2BYRHCJ3c35Gv4SmihuGVXUS3WFsL0i3VyXno8xPOfmyuUT7GZWW80uMnV6eOyhbPCBb17CchI8x7owdXVf3drxCrrjoP3oNbbpByTsDC+hnCmVQbhkZAqS63gZRj5LT6gNh4ZA/qenaGkhzfHFRSUW1+H16ZzYcSSl0PfAQBQYrc7agUjI987PBHdOv6BJS+pKmZ0MvaDwbnzhHZZdb8LXqM5K0zAhPiZMPL2MTq7+eNOBdRuTlbWs3OZfbY1cnewSpMmUGfwrIjVnLtvfhfJXVaFbpIfvNc6eafgmHas23A05f4Gk/IcRLvqICw+GQg76ADUpWubQal7gECTGbeivOdRLnQcQYWCg/a5VR5B+4ELENmWLRD7IQgo6rJKpPiRdGRc0C0E9Mqt3w7w+9NiDnjTbSu8Q6ARBLXjAu+jSbY1VK0iPq6k1Wj8U5EgQF/xVLCEtKDRW1bdllwWNgi2yPo1CrkgKmJj6XPU3Zc744NR85uC+PxAjkIIbm90joevPmUQGwkpAgVX2pxCjVAX6ynQ79TFfdd/Y=
 
+sudo: required
```
Comment (ppolewicz) [inline .travis.yml]: why?

Comment (szepeviktor) [inline .travis.yml]: OK. We try the container based infrastructure which comes without sudo.

Comment (ppolewicz) [inline .travis.yml]: It is much better to use the containers whenever possible

**Comment by szepeviktor** [comment]
OK.
*Green* to merge.

**Comment by ppolewicz** [comment]
@bwbeach ready to merge

**Pull request merged**